react.md

```diff
@@ -4,30 +4,28 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. Here is a list of pros and cons to using the React library to build your application -- but some of them are false. Remove the false statements from the list:
 
-- React was created to be simple, so that even people with minimal code experience could use it and create Single Page Applications (SPAs)
 - React is a modern, efficient answer to complex UI applications
-- React is a full stack framework for modern web applications
-- React is a flexible library that plays the role of V in an MVC framework
+- React is a full stack framework for modern web application- React is a flexible library that plays the role of V in an MVC framework
+
 
- 
  #### 2. What are "smart"(logic) and "dumb"(display) components? Explain the difference and also add why we bother to make the distinction between them.
- 
- 
+
  //Your Answer
- 
- 
+ Smart components have functions and logic as part of the component.  Dumb components do not have any functions.
+
+
  //Googled Answer
- 
- 
+Smart components are app level components that perform functions and manage data while dumb components focus solely on the UI
+
 #### 3. When we use "yarn add ..." in the terminal - what is yarn doing? And what file will always be automatically updated after we add a package with yarn?
- 
- 
+
+
  //Your Answer
- 
- 
+Yarn is a respository for files.  Yarn is adding a file to the your database.  Javascript will be updated.
```

Comment:
This is essentially correct. Yarn is a package manager that adds node packages to your applications. Revise this answer to include what file it is that the yarn add command updates. (Hint: run 'yarn add react-bootstrap' in a React app and find where that package name gets added)